### PR TITLE
Issue 912

### LINKED ISSUE #912: Weird behavior when creating a new project while the constraint view is closed
#### Prerequisitives
* **Eclipse Version:** 2019-03
* **Operating system:** Windows 10
* **Git branch/Commit-ID:**  release3.6 https://github.com/FeatureIDE/FeatureIDE/commit/6173d671966b5824f4beff3748b8cb7b1e306dd4

### Issue description
Closed the constraints view and tried to create a new feature modeling project. When pressing finish, there is a popup as for the newly created project the feature model is opened.

Pressing abort seems to do nothing. Pressing no seems to work fine. Pressing yes results in some weird behavior that I first need to press finish again in the project creation wizard and then need to answer the other dialog. Maybe it would help to first close the old dialog before proceed opening the feature model.

### PR DESCRIPTION
**Pull request by h3ssto**
Fixed #912

### CODE REVIEW

File: plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/util/ConstraintViewDialog.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* FeatureIDE - A Framework for Feature-Oriented Software Development
- * Copyright (C) 2005-2019  FeatureIDE team, University of Magdeburg, Germany
+ * Copyright (C) 2005-2017  FeatureIDE team, University of Magdeburg, Germany
```
Comment (PaulWestphal) [inline plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/util/ConstraintViewDialog.java:2]: ?

File: plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/util/ConstraintViewDialog.java [hunk 2]
```diff
@@ -20,187 +20,44 @@
...
 
 import de.ovgu.featureide.fm.core.Preferences;
 import de.ovgu.featureide.fm.core.localization.StringTable;
-import de.ovgu.featureide.fm.ui.views.constraintview.ConstraintViewController;
 
 /**
  * A dialog that asks a user to open the constraint view. Its saves the decision in the workspace wide preferences.
  *
- * @author Rosiak Kamil
+ * @author Rosiak Kamil (no functionality of that implementation left)
```
Comment (PaulWestphal) [inline plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/util/ConstraintViewDialog.java:35]: Remove the name or the "(no functionality of that implementation left)".

File: plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/util/ConstraintViewDialog.java (hunk 1 above)
Comment (h3ssto) [reply plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/util/ConstraintViewDialog.java:2]: Makro magic.

File: plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/util/ConstraintViewDialog.java (hunk 2 above)
Comment (h3ssto) [reply plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/util/ConstraintViewDialog.java:35]: Ok.

**Pull request merged**